src/components/VersionRestartModal/TaskStatusCheckboxContainer.tsx

```diff
@@ -24,7 +26,7 @@ export const TaskStatusCheckboxContainer: React.FC<TaskStatusCheckboxContainerPr
   const toggleHandler = (e) => {
     const { name } = e.target;
     if (selectedTasks[name] !== undefined) {
-      toggleSelectedTask(name);
+      toggleSelectedTask({ [versionId]: name });
```

Comment:
Would it make sense for this to only accept an array of taskIds for clarity?
```suggestion
      toggleSelectedTask({ [versionId]: [name] });
```

Reply:
I think that would make it more complicated than necessary. This part of the logic predates me and is already pretty well defined.